extensions/amp-consent/0.1/test-e2e/test-amp-consent-server-side.js

```diff
@@ -79,7 +78,11 @@ describes.endtoend(
         'postPromptUi': false,
       });
 
-      // Check the analytics request consentState
+      // Check the analytics request consentState. Wait for 1 second for the
+      // request to arrive to avoid flaky test.
+      await new Promise((resolve) => {
+        setTimeout(resolve, 1000);
+      });
```

Comment:
I believe in `test-amp-consent-server-side-expire-cache.js`we are waiting 3 seconds instead of 1, do you think 1 second will suffice here?

Reply:
Ha didn't know we have `await sleep()`, I'll switch to that. 
1 sec seemed to work fine. Any reason 3 seconds is chosen? We can always extend later.

Reply:
I think you'll have to import it though! It's called `sleep-promise`

Reply:
oops...

Reply:
iirc, @estherkim found 3 seconds to be the most reliable for that test

Reply:
ok, switched to 3secs